vite.config.ts

```diff
@@ -6,7 +6,7 @@ import tsconfigPaths from "vite-tsconfig-paths";
 // for deployments that are deployed to a sub-folder like /dev, specify below or use the `BASE_URL` env var
 const DEFAULT_BASE_URL = "/";
 // to proxy transaction api requests on a different domain, specify below or use the `PROXY_URL` env var
-const DEFAULT_PROXY_URL = "https://n2ft4ng3qh.execute-api.us-east-1.amazonaws.com/dev";
+const DEFAULT_PROXY_URL = "https://719hgl5ejk.execute-api.us-east-1.amazonaws.com/dev";
```

Comment:
Non-blocking, wouldn't it better to parameterize this out as env var or something to churn with commits and also not keep this in the repo?

(NOTE: is the intention of this repo to be public later at a yet to be determined date?)

Reply:
Resolved in the new commit